crates/core/src/db/queries/metadatas.rs

```diff
@@ -63,9 +63,7 @@ pub fn list(
         .inner_join(
             token_accounts::table.on(metadatas::mint_address.eq(token_accounts::mint_address)),
         )
-        .left_outer_join(
-            listing_receipts::table.on(metadatas::address.eq(listing_receipts::metadata)),
-        )
```

Comment:
This inner_join will require that each metadata has a listing_receipt which isn't the case so all the metadata that don't have a receipt will be tossed out.

Reply:
What we need to be able to do is add the join conditionally with the filter so if you put that you want nfts that are listed we can do inner join for all.